spec/models/manageiq/providers/google/cloud_manager/refresher_spec.rb

```diff
@@ -135,6 +143,27 @@ def assert_specific_availability_zone
     )
   end
 
+  def assert_specific_cloud_database_flavor
+    expect(cloud_database_flavor).to have_attributes(
+      :name    => "db-f1-micro",
+      :type    => "ManageIQ::Providers::Google::CloudManager::CloudDatabaseFlavor",
+      :ems_ref => "db-f1-micro",
+      :memory  => 644_245_094,
+      :enabled => true
+    )
+  end
+
+  def assert_specific_cloud_database
+    expect(cloud_database).to have_attributes(
+      :name                  => "miq-test",
+      :type                  => "ManageIQ::Providers::Google::CloudManager::CloudDatabase",
+      :ems_ref               => "miq-test",
+      :db_engine             => "POSTGRES_13",
+      :status                => "RUNNABLE",
```

Comment:
Not for this PR, but is there any normalization for these values so we can display it consistently?